.chronus/changes/oa3-to-ts-2024-5-26-13-27-36.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: feature
+packages:
+  - "@typespec/openapi3"
+---
+
+Adds support for converting OpenAPI3 specs to TypeSpec via the new openapi3-to-tsp CLI included in the @typespec/openapi3 package.
```

Comment:
One suggestion about the usage, can we make the CLI similar to the openapi2 to TypeSpec converter by integrating it into tsp-client?
https://azure.github.io/typespec-azure/docs/howtos/migrate-swagger/get-started

Reply:
Are you suggesting aligning on CLI conventions (e.g. `convert` instead of `compile`)? If so, seems reasonable. Or are you suggesting putting this code into @azure-tools/typespec-client-generator-cli and 3p folks use that instead? I would not support that.  But I'm sure it wouldn't be difficult to wrap this CLI in that tool if we wanted to.

Reply:
FYI `tsp-client` https://github.com/Azure/azure-sdk-tools/tree/main/tools/tsp-client

Reply:
+1 on naming the command `convert` (or just not having one as the cli name already is descrptive)
but same against moving this to tsp client which is an azure tool to work with our multi repo setups

Reply:
Updated the naming conventions and this changelog.